src/modules/extrakeys/tests_impl.h

```diff
@@ -260,33 +245,32 @@ void test_xonly_pubkey_tweak_check(void) {
     unsigned char tweak[32];
 
     int ecount;
-    secp256k1_context *none = api_test_context(SECP256K1_CONTEXT_NONE, &ecount);
-    secp256k1_context *sign = api_test_context(SECP256K1_CONTEXT_SIGN, &ecount);
-    secp256k1_context *verify = api_test_context(SECP256K1_CONTEXT_VERIFY, &ecount);
+
+    set_counting_callbacks(ctx, &ecount);
```

Comment:
Here's an example of #1167 in the wild. (maybe that's how you found this issue?)

Anyway, we have this pattern many times in the tests. This needs separate solution that should not hold off this PR.